packages/generator/src/v1.1/backcompat/internal-foundry-ontology-runtime-dist/generateFiltersDir.ts

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import path from "node:path";
+import type { MinimalFs } from "../../../MinimalFs";
+import { formatTs } from "../../../util/test/formatTs";
+import { reexportConsts } from "../util/reexportConsts";
+import { reexportTypes } from "../util/reexportTypes";
+
+export async function generateFiltersDir(
+  runtimeDistDir: string,
+  fs: MinimalFs,
+) {
+  const pagingDir = path.join(runtimeDistDir, "filters");
+  await fs.mkdir(pagingDir, { recursive: true });
+
+  await fs.writeFile(
+    path.join(pagingDir, "index.ts"),
+    await formatTs(
+      `export * from "./ArrayFilter";
+      export * from "./AttachmentFilter";
+      export * from "./BooleanFilter";
+      export * from "./DateTimeFilters";
+      export * from "./Filters";
+      export * from "./GeoPointFilter";
+      export * from "./GeoShapeFilter";
+      export * from "./NumericFilter";
+      export * from "./Op";
```

Comment:
You might have to re-const the Op stuff to make it useable.

Reply:
It's re-exported as a const in `./Op.ts`, and that seems to work